package.json

```diff
@@ -5,12 +5,19 @@
     "codegen": "graph codegen",
     "build": "graph build",
     "deploy": "graph deploy --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ salazarguille/zero-collateral",
-    "create-local": "graph create --node http://localhost:8020/ salazarguille/zero-collateral",
-    "remove-local": "graph remove --node http://localhost:8020/ salazarguille/zero-collateral",
-    "deploy-local": "graph deploy --node http://localhost:8020/ --ipfs http://localhost:5001 salazarguille/zero-collateral"
+    "create-local": "graph create --node http://127.0.0.1:8020/ salazarguille/zero-collateral",
+    "remove-local": "graph remove --node http://127.0.0.1:8020/ salazarguille/zero-collateral",
+    "deploy-local": "graph deploy --node http://127.0.0.1:8020/ --ipfs http://localhost:5001 salazarguille/zero-collateral",
```

Comment:
❓ wasn't this fix?

Reply:
Yes, it was fixed in the PR #5 
The subgraph was transferred after this PR.